guides/v2.1/cloud/cdn/cloud-vcl-custom-snippets.md

```diff
@@ -3,7 +3,7 @@ group: cloud-guide
 subgroup: 090_configure
 title: Custom Fastly VCL snippets
 menu_title: Custom Fastly VCL snippets
-menu_order: 7
+menu_order: 40
```

Comment:
are we using this (menu_ stuff)? I thought these were used for tutorial templates. I could be wrong.

Reply:
I will check on this. I tried without the ordering and was getting build errors, which went away when I added the ordering. The navigation renders OK, so I think we can merge this, and remove the ordering based on what I learn about how it works.